package.json

```diff
@@ -64,7 +64,7 @@
     "json-loader": "^0.5.4",
     "jstransformer-markdown-it": "^2.0.0",
     "lolex": "^1.5.2",
-    "node-sass": "^4.9.3",
+    "node-sass": "^6.0.0",
```

Comment:
Hey @mattelen! Have you checked if there are any possible breaking changes since those are 2 major versions?

Reply:
Ahh I miss this commit! I had issues getting this version of node-sass working on my local, but realised that my node version was too high. I've reverted it as it now works fine. Thanks for picking this up!